ui/app/templates/components/secret-edit-toolbar.hbs

```diff
@@ -92,12 +92,15 @@
       {{#let (concat "vault.cluster.secrets.backend." (if (eq @mode "show") "edit" "show")) as |targetRoute|}}
         {{#if @isV2}}
           <ToolbarLink
-            @params={{array targetRoute @model.id (query-params version=@modelForData.version)}}
+            {{! Always create new version from latest if no metadata read access }}
```

Comment:
Is this case even if you're previewing an older version of the secret?

Reply:
Right now through the UI there's not a way to view an older version if you don't have metadata read access, but if there's a use case there then we can calculate whether to include the version based on whether it's on the query param or if they can read metadata

Reply:
Got it!